Source/WebCore/css/BasicShapesShapeSegmentConversion.cpp

```diff
@@ -54,6 +54,14 @@ static auto lengthSizeToCSSValue(const LengthSize& value, const RenderStyle& sty
         CSSPrimitiveValue::create(value.height, style));
 }
 
+static auto controlPointToControlPointValue(const ControlPoint& controlPoint, const RenderStyle& style)
```

Comment:
We don't use `auto` for return type for this case generally

Reply:
Buy _why_ don’t we?

I think we can just name this function `controlPointValue` and while others rarely choose this style, I might even name it just `value` since it’s local and unambiguous due to the argument type. Don’t need to say the name of an argument type, and the shorter name feels easier to read.